pkg/controller/migmigration/backup.go

```diff
@@ -42,14 +44,20 @@ func (t *Task) ensureInitialBackup() (*velero.Backup, error) {
 	if err != nil {
 		return nil, liberr.Wrap(err)
 	}
+	userIncludedResources, err := t.getUserIncludedResourceList(client)
+	if err != nil {
+		return nil, liberr.Wrap(err)
+	}
 
 	newBackup.Labels[migapi.InitialBackupLabel] = t.UID()
 	newBackup.Labels[migapi.MigMigrationDebugLabel] = t.Owner.Name
 	newBackup.Labels[migapi.MigPlanDebugLabel] = t.Owner.Spec.MigPlanRef.Name
 	newBackup.Labels[migapi.MigMigrationLabel] = string(t.Owner.UID)
 	newBackup.Labels[migapi.MigPlanLabel] = string(t.PlanResources.MigPlan.UID)
 	newBackup.Spec.IncludedResources = toStringSlice(settings.IncludedInitialResources.Difference(toSet(t.PlanResources.MigPlan.Status.ExcludedResources)))
+	newBackup.Spec.IncludedResources = append(newBackup.Spec.IncludedResources, userIncludedResources...)
```

Comment:
Note that the default value for IncludedInitialResources is an empty list, meaning all resources. If you add something here, it's essentially a subtraction, saying "everything not returned by `getUserIncludedResourceList` is excluded.